internal/blockchain/common/blockchain.go

```diff
@@ -0,0 +1,152 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
```

Comment:
Would it make sense to put this in `internal/blockchain/common.go` instead?

Reply:
That's what I originally had, but wasn't sure. I think `common` does make more sense, since the file contains a mix of `callbacks` and other blockchain pieces.